### PR TITLE
net-misc/gns3-server (net-misc/gns3-gui): bump to 2.2.0

### PR DESCRIPTION
**Pull request by linxon**
1) Bump `net-misc/gns3-server` and `net-misc/gns3-gui` to `2.2.0`
2) Added new `dev-python/aiofiles` ebuild (req `net-misc/gns3-server`)
3) `dev-python/distro` bump

Bugs:
https://bugs.gentoo.org/672882
https://bugs.gentoo.org/693400
https://bugs.gentoo.org/688016

Signed-off-by: Yury Martynov email@linxon.ru

### CODE REVIEW

File: dev-python/aiofiles/aiofiles-0.4.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+inherit distutils-r1
+
+DESCRIPTION="File support for asyncio"
+HOMEPAGE="https://pypi.org/project/aiofiles/"
+SRC_URI="mirror://pypi/${P:0:1}/${PN}/${P}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+IUSE=""
```
Comment (ConiKost) [inline dev-python/aiofiles/aiofiles-0.4.0.ebuild]: If empty, drop it.

File: dev-python/aiofiles/aiofiles-0.4.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+RESTRICT=""
```
Comment (ConiKost) [inline dev-python/aiofiles/aiofiles-0.4.0.ebuild]: If empty, drop it.

**Comment by linxon** [comment]
>www.gns3.net’s server IP address could not be found.

this website is not accessible for me. Maybe I need try to change a HOMEPAGE to the github web page?

**Comment by ConiKost** [comment]
> this website is not accessible for me. Maybe I need try to change a HOMEPAGE to the github web page?

Is there an offical github repo? If so, I would add it.

**Comment by juippis** [comment]
3e6f81303185dde8d59a9696b978b12f4196af40

**Pull request closed without merging**